tests/test_fieldset.py

```diff
@@ -905,3 +908,30 @@ def test_fieldset_from_data_gridtypes(xdim=20, ydim=10, zdim=4):
     pset.execute(AdvectionRK4, runtime=1, dt=.5)
     assert np.allclose(plon, pset.lon)
     assert np.allclose(plat, pset.lat)
+
+
+@pytest.mark.parametrize('mode', ['scipy', 'jit'])
+@pytest.mark.parametrize('direction', [1, -1])
+@pytest.mark.parametrize('time_extrapolation', [True, False])
+def test_deferredload_simplefield(mode, direction, time_extrapolation, tmpdir, tdim=10):
```

Comment:
I assume that's also the test to see that the non-dask `DeferredNetcdfFileBuffer` also works as-expected. Nice.